src/components/seleced-film/SelectedFilm.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import './SelectedFilm.css';
+import { Link } from 'react-router-dom';
+
+class SelectedFilm extends Component {
+  render() {
+    return (
+      <nav className="navbar navbar-expand-md navbar-dark bg-dark header">
+        <div className="container">
+          <div className="row mt-1 w-100">
+            <div className="col-4">
+              <span className="pink font-weight-bold">netflixroulette</span>
+            </div>
+            <div className="col-auto ml-auto">
+              <Link to="/">
+                <button className="btn my-2 my-sm-0 white button-pink" type="button">
```

Comment:
button inside link looks too strange.. It should be a button with onClick handler or link with button styles.